src/utils_stackdriver_json_test.c

```diff
@@ -71,9 +71,12 @@ static int run_summary_test(time_series_summary_t summary) {
 
 DEF_TEST(time_series_summary) {
   char buf[10000];
+  ssize_t file_size;
   int ret;
-  OK(read_file_contents(SRCDIR "/src/time_series_summary_test.json", buf,
-                        sizeof(buf)) >= 0);
+  file_size = read_file_contents(SRCDIR "/src/time_series_summary_test.json", buf,
+                                 sizeof(buf));
```

Comment:
This needs to be `sizeof(buf) - 1`, else you're at a risk of buffer overrun in line 79.

Reply:
Addressed by #174.